tests/unit/decoder/src/test_decode_manifest.c

```diff
@@ -365,15 +365,93 @@ static uint8_t cbor_envelope_with_6_components[] = {
 			0x40, /* bytes(0) */
 
 	0x03, /* suit-manifest */
-	0x58, 0x1c, /* bytes(28) */
+	0x58, 0x2f, /* bytes(37) */
 	0xa3, /* map (3 elements) */
 	0x01, /* suit-manifest-version */ 0x01,
 	0x02, /* suit-manifest-sequence-number */ 0x10,
 	0x03, /* suit-common */
-		0x55, /* bytes(21) */
+		0x58, 0x27, /* bytes(30) */
 		0xA1, /* map (1 element) */
 			0x02, /* suit-components */
-				0x86, /* array (6 elements) */
+				0x8c, /* array (13 elements) */
+				0x81, /* array (1 element) */
+					0x41, /* bytes(1) */
+					'M',
+				0x81, /* array (1 element) */
+					0x41, /* bytes(1) */
+					'M',
+				0x81, /* array (1 element) */
+					0x41, /* bytes(1) */
+					'M',
+				0x81, /* array (1 element) */
+					0x41, /* bytes(1) */
+					'M',
+				0x81, /* array (1 element) */
+					0x41, /* bytes(1) */
+					'M',
+				0x81, /* array (1 element) */
+					0x41, /* bytes(1) */
+					'M',
+				0x81, /* array (1 element) */
+					0x41, /* bytes(1) */
+					'M',
+				0x81, /* array (1 element) */
+					0x41, /* bytes(1) */
+					'M',
+				0x81, /* array (1 element) */
+					0x41, /* bytes(1) */
+					'M',
+				0x81, /* array (1 element) */
+					0x41, /* bytes(1) */
+					'M',
+				0x81, /* array (1 element) */
+					0x41, /* bytes(1) */
+					'M',
+				0x81, /* array (1 element) */
+					0x41, /* bytes(1) */
+					'M',
+};
+
+static uint8_t cbor_envelope_with_13_components[] = {
+	0xd8, 0x6b, /* tag(107) : SUIT_Envelope */
+	0xa2, /* map (2 elements) */
+
+	0x02, /* suit-authentication-wrapper */
+		0x42, /* bytes(2) */
+		0x81, /* array (1 element) */
+			0x40, /* bytes(0) */
+
+	0x03, /* suit-manifest */
+	0x58, 0x32, /* bytes(37) */
+	0xa3, /* map (3 elements) */
+	0x01, /* suit-manifest-version */ 0x01,
+	0x02, /* suit-manifest-sequence-number */ 0x10,
+	0x03, /* suit-common */
+		0x58, 0x2a, /* bytes(30) */
```

Comment:
`0x2a` is not `30`

Reply:
The comment was wrong, updated the value.